internals/generators/container/stateless.js.hbs

```diff
@@ -46,7 +55,7 @@ function {{ properCase name }}() {
 
 {{#if wantActionsAndReducer}}
 const mapStateToProps = createStructuredSelector({
-  {{name}}: makeSelect{{properCase name}}(),
+  {{ lowerCase name }}: makeSelect{{properCase name}}(),
```

Comment:
Unrelated, so not for this PR, but while we're touching this, can I ask why this was change to use createStructuredSelector here instead of directly call makeSelect, as it was before IIRC? I've been finding myself removing this every time I generate a container.